server/tests/users.test.js

```diff
@@ -1,14 +1,21 @@
 import chai from 'chai';
 import chaiHttp from 'chai-http';
 import app from '../app';
+import generateToken from './factory/user-factory';
 
 chai.use(chaiHttp);
 const { expect } = chai;
 
 // define the token variable
-let authToken;
+let userToken;
 
 describe('TESTS TO SIGNUP A USER', () => {
+  before(async () => {
+    userToken = await generateToken({
```

Comment:
This works

However, I don't want you to hardcode the user values.

in the `before` hook:

- You should generate a `new user` (Using `faker`)
- You should generate the `userToken` (as you have currently done but using the user details created from `faker`

Reply:
I studied `faker` docs and found how to implement the user generation just like you pointed out. the changes have been implemented in the new pull request and the test works just fine.

Reply:
it seems you only partially understand what I meant.

- Create a NEW USER in this `before` hook using `faker`
- generate token from that NEW USER

As it stands, when we merge #20 and this ticket, the system would run into problems

Reply:
Noted